lusidtools/cocoon/cocoon.py

```diff
@@ -1426,6 +1429,15 @@ def load_from_data_frame(
         .value
     )
 
+    Validator(
+        property_columns, "property_columns"
+    ).check_entries_are_strings_or_dict_containing_key("source")
+
+    property_columns = [
+        {"source": column, "target": column} if isinstance(column, str) else column
+        for column in property_columns
```

Comment:
What happens if I pass a list with

property_columns = [
 {"source": MyColumnName},
 {"source": MySecondColumnName}
]

Reply:
i.e. missing the "target" key

Reply:
Never mind, I can see how this is handled, looks good.